src/task/ReceiveAudioInputTask.ts

```diff
@@ -15,11 +15,16 @@ export default class ReceiveAudioInputTask extends BaseTask {
   }
 
   async run(): Promise<void> {
+    if (!this.context.meetingSessionConfiguration?.urls?.audioHostURL?.length) {
+      this.context.logger.info('no audio connection');
```

Comment:
Do we need to log this?

Reply:
I think it's the only place that no audio connection case is logged, would it be better if I log it in audio video controller ?